Sources/Document/Change/ChangeID.swift

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2022 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/**
+ * `ChangeID` is for identifying the Change. This is immutable.
+ */
+struct ChangeID {
+    /**
+     * `initial` represents the initial state ID. Usually this is used to
+     * represent a state where nothing has been edited.
+     */
+    static let initial = ChangeID(clientSeq: 0, lamport: 0, actor: ActorIDs.initial)
+
+    // `serverSeq` is optional and only present for changes stored on the server.
+    private var serverSeq: Int64?
```

Comment:
@hackerwins This is not used. Can I remove it?

Reply:
Because it can be used for the restore feature to revert to a specific change, I think it would be good to leave it for now.
https://github.com/yorkie-team/yorkie/issues/52

Reply:
OK, I'll leave it.